templates/taxi/manufacturer_list.html

```diff
@@ -1,33 +1,14 @@
 {% extends "base.html" %}
+{% load crispy_forms_filters %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+  <h1>{{ object|yesno:"Update, Create" }} Create a Manufacturer</h1>
```

Comment:
The use of the `yesno` filter with `object` is incorrect here. The `yesno` filter is typically used with boolean values to display different text based on the value. Ensure that `object` is a boolean or adjust the logic to correctly display 'Update' or 'Create' based on the context.